packages/default/scss/toolbar/_layout.scss

```diff
@@ -158,6 +158,11 @@
             &:not(:last-child) {
                 margin-right: $toolbar-padding-x;
             }
+
+            .k-separator {
+                margin: 0;
```

Comment:
do not reset top and bottom margin. use `margin-left` and `margin-right` instead